bin/node/runtime/src/lib.rs

```diff
@@ -1306,15 +1358,24 @@ impl_runtime_apis! {
 		}
 	}
 
+	#[cfg(feature = "try-runtime")]
```

Comment:
[Testing tool](https://docs.substrate.io/reference/command-line-tools/try-runtime/#:~:text=The%20try%2Druntime%20tool%20is,state%20before%20going%20to%20production.)